code/game/objects/items/weapons/grenades/ghettobomb.dm

```diff
@@ -16,6 +16,10 @@
 	display_timer = FALSE
 	var/list/times
 
+/obj/item/grenade/iedcasing/examine(mob/user)
+	. = ..()
+	. += "<span class='warning'>You have no idea how long the fuze will last for until it explodes!</span>"
```

Comment:
p sure we prefer american english for text like this
```suggestion
	. += "<span class='warning'>There's no way to tell how long the fuse will last for until it explodes!</span>"
```

Reply:
Fuze is American English for the explosive timing device (with fuse being preferred for the flammable cord). As seen on the [relevant page of that one encyclopedia](https://en.wikipedia.org/wiki/Fuze). When googling "Fuze vs Fuse" the first thing that came up was a US marine corps powerpoint that also expressed this sentiment.